src/scripts/main.js

```diff
@@ -1,3 +1,94 @@
 'use strict';
 
-// write code here
+const getTableHeader = document.querySelectorAll('table th');
+const getTBody = document.querySelectorAll('tbody tr');
+
+getTableHeader.forEach((header) => {
+  // eslint-disable-next-line no-shadow
+  header.addEventListener('click', function (event) {
+    const getClickOnHeader = event.target.closest('th');
+
+    if (getClickOnHeader && header.textContent === 'Name') {
+      const arrForName = [];
+
+      // Для імен
+      getTBody.forEach((itemName) => {
+        const getCellersName = itemName.cells[0];
+
+        arrForName.push(getCellersName.textContent);
+      });
+
+      arrForName.sort();
+
+      // Оновлюємо першу колонку таблиці
+      getTBody.forEach((itemName, index) => {
+        const getCellersName = itemName.cells[0];
+
+        getCellersName.textContent = arrForName[index] || '';
+      });
+    }
+
+    if (getClickOnHeader && header.textContent === 'Position') {
+      // Для позиції
+      const arrPosition = [];
+
+      getTBody.forEach((itemPosition) => {
+        const getCellersPosition = itemPosition.cells[1];
+
+        arrPosition.push(getCellersPosition.textContent);
+      });
+
+      arrPosition.sort();
+
+      getTBody.forEach((itemPosition, index) => {
+        const getCellersPosition = itemPosition.cells[1];
+
+        getCellersPosition.textContent = arrPosition[index] || '';
+      });
+    }
+
+    if (getClickOnHeader && header.textContent === 'Age') {
+      const arrAge = [];
+
+      getTBody.forEach((itemAge) => {
+        const getCellersAge = itemAge.cells[2];
+
+        arrAge.push(getCellersAge.textContent);
+      });
+
+      arrAge.sort();
```

Comment:
The sorting for 'Age' is currently done as a string. This may lead to incorrect sorting results for numerical values. Consider converting the age values to numbers before sorting.